lua/neo-tree/sources/filesystem/commands.lua

```diff
@@ -94,6 +94,9 @@ end
 M.open_vsplit = function(state)
   cc.open_vsplit(state, utils.wrap(fs.toggle_directory, state))
 end
+M.open_tabnew = function (state)
```

Comment:
This command should also be added to the `buffers` and `git_status` sources to be consistent with the other open commands.

Reply:
Done